keyboards/lets_split/keymaps/piemod/keymap.c

```diff
@@ -93,29 +93,8 @@ enum function_id {
 int CAPSLOCKED = 0;
 
   const uint16_t PROGMEM fn_actions[] = {
-
-    // DVORAK 0
-    // ARROW 1, F(1)
-    // SYMBOL 2, F(2)
-    // NUMBER 3, F(3)
-    // FUNCTION 4, F(4)
-    // EMACS 5, F(5)
-    // MACROS 6, F(6)
-    // MOUSE 7, F(7)
-
-    // Layers
-    [1] =  ACTION_LAYER_TAP_KEY(1, KC_A),     // FN1 = Momentary Arrow layer on A.
-    [2] =  ACTION_LAYER_TAP_KEY(2, KC_O),     // FN2 = Momentary symbOl layer on O.
-    [3] =  ACTION_LAYER_TAP_KEY(3, KC_E),     // FN3 = Momentary numbEr layer on E.
-    [4] =  ACTION_LAYER_TAP_KEY(4, KC_U),     // FN4 = Momentary fUnction layer on U.
-    [5] =  ACTION_LAYER_TAP_KEY(5, KC_Q),     // FN5 = Momentary emaQs layer on Q.
-    [6] =  ACTION_LAYER_TAP_KEY(6, KC_LCTRL), // FN6 = Momentary MACROS on ??? key.
-    [7] =  ACTION_LAYER_TAP_KEY(7, KC_COMMA), // FN7 = Momentary MOUSE on , key.
-
     // Special Keys
-    [10] =  ACTION_MODS_KEY(KC_LCTRL, KC_LALT), // FN10 = Ctrl + Alt.
-
+    [1] =  ACTION_MODS_KEY(KC_LCTRL, KC_LALT), // FN10 = Ctrl + Alt.
```

Comment:
If these are doing what I think they're doing, then `MT(mod, kc)` handles this as well: https://docs.qmk.fm/features/advanced-keycodes#mod-tap

And it would be `MT(MOD_LCTL, KC_LALT)` and `MT(MOD_LSFT, KC_SLSH)` respectively. 

No need to change this, unless you want.

Reply:
MT is another really cool addition I wasn't familiar with, but I don't think it's a drop in for this case. The functions are supposed to call both keys at the same time (e.g., to get a question mark for the latter).

If it's OK with you, I will follow-up with another pull request with some more functionality related changes.

Reply:
An, in that case, you'd just want these: https://docs.qmk.fm/keycodes#us-ansi-shifted-keys